src/routes.py

```diff
@@ -112,6 +112,42 @@ def device_create():
     return jsonify(res), 200
 
 
+@api.route('device/edit', methods=["PATCH"])
+@requires_auth
+def device_edit():
+    """Route for a user to edit a device values."""
+    device_data = request.get_json()
+    ID_device = device_data["DeviceID"]
+    new_area = float(device_data["NewArea"])
```

Comment:
```suggestion
    new_area = int(device_data["NewArea"])
```

Ainda precisa mudar isso aqui, Gi!